app/institutions/dashboard/projects/controller.ts

```diff
@@ -0,0 +1,73 @@
+import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+import Intl from 'ember-intl/services/intl';
+
+import { ResourceTypeFilterValue } from 'osf-components/components/search-page/component';
+import { ObjectListColumn } from '../-components/object-list/component';
+
+export default class InstitutionDashboardProjects extends Controller {
+    @service intl!: Intl;
+
+    columns: ObjectListColumn[] = [
+        { // Title
+            name: this.intl.t('institutions.dashboard.object_list.table_headers.title'),
+            valuePath: 'displayTitle',
+        },
+        { // Link
+            name: this.intl.t('institutions.dashboard.object_list.table_headers.link'),
+            type: 'link',
+            hrefValuePath: 'indexCard.osfUrl',
+            linkText: 'indexCard.guidFromIdentifierList',
+        },
+        { // Object type
+            name: this.intl.t('institutions.dashboard.object_list.table_headers.object_type'),
+            valuePath: 'intlResourceType',
+        },
+        { // Date created
+            name: this.intl.t('institutions.dashboard.object_list.table_headers.created_date'),
+            valuePath: 'dateFields.0.date', // this feels quite brittle...
```

Comment:
agree, better to get `dateCreated` from the metadata

Reply:
Yea, was intending to address this, but forgot about it in my pile of `TODOs` in this PR 😅 . Seems best to get the date field explicitly from the resourceMetadata without being bound by the assumptions in the `SearchResultCard` component (where this `dateFields` property is used)